addons/web/static/tests/views/fields/many2many_tags_field.test.js

```diff
@@ -1527,23 +1839,23 @@ test("Many2ManyTagsField selected records still pickable and not duplicable", as
             </list>
         `,
     });
-
+    const item = getMockEnv().isSmall ? ".o_kanban_record" : "a.dropdown-item";
```

Comment:
I think this is a good candidate for having both desktop and mobile version of this test